src/components/hocs/db/withSpaceById.ts

```diff
@@ -15,7 +15,7 @@ export const withSpaceById = <T extends Attributes>(
 ) => {
   const WithSpaceById = (props: Props<T>) => {
     const { space, spaceId, worldId, isLoaded, isLoading } = useSpaceById(
-      props
+      props.spaceId
     );
```

Comment:
Please don't change the `useSpaceById` signature. Accepting `options` object with `spaceId` field is providing for backward and forward compatibility in ways a single string parameter can't.

Reply:
[8f7d0bc](https://github.com/sparkletown/sparkle/pull/2964/commits/8f7d0bcb22aad901b7d6456cb7b36d6787fe4ac1)